Shopify/shopify-cli
# Remove AssemblyScript

### WHY are these changes introduced?

We are removing AssemblyScript as a supported script language.

Related issue: https://github.com/Shopify/script-service/issues/3894

### WHAT is this pull request doing?

Removes references to AssemblyScript.

### How to test your changes?

- For both Wasm and TypeScript, the following operations should succeed:
  1. Create a script
  2. Push the script

- For AS, attempt to create a project using the `--language assemblyscript` flag; it shouldn't work.

- In an existing project, attempt to push; it shouldn't work.

### Post-release steps

Once this is released the backend cleanup in the linked ticket can be shipped.

### Update checklist
